### DISCUSSION TITLE
Does cpptools support set value of register?

### DISCUSSION THREAD

**Discussion by wss29** [microsoft/vscode-cpptools#12424] [Q&A]
hi,
    When I right-click on local variable I can set value of it, But when I right-click on the register, I can not set the value of register, Does cpptools support set value of register? I use gdb as debugger.  I have try it on cdt-gdb-adapter, cdt-gdb-adapter can set value of register
![image](https://github.com/microsoft/vscode-cpptools/assets/4537809/24663990-8793-4cae-af65-a903a0918bd5)

**Comment by sean-mcmanus** [comment]
@wss29 No. It's tracked by https://github.com/microsoft/MIEngine/issues/1099
